assets/dev/js/editor/regions/navigator/navigator.js

```diff
@@ -92,7 +92,11 @@ export default class extends BaseRegion {
 
 		this.$el.draggable( this.getDraggableOptions() );
 
-		this.$el.resizable( this.getResizableOptions() );
+		// Used timeout here since draggable and resizable are conflicting.
```

Comment:
This issue didn't exist in the past. It's probably due to another change that screwed it up

Reply:
@iNewLegend , try chaining, like here:
https://stackoverflow.com/a/16990819/5503355

Reply:
> This issue didn't exist in the past. It's probably due to another change that screwed it up
Yes ( i tested in 2.7.5 ).


> @iNewLegend , try chaining, like here:
> https://stackoverflow.com/a/16990819/5503355

tried didn't resolve the issue.

Reply:
Nope. This issue for sure didn't exist before.

Reply:
The issue was caused beacuse of `$e.route( 'navigator' )`, i don't know why but when `initLayout` was called via `$e.route` resize able become screwed.

tested on document change resize is working.
https://github.com/elementor/elementor/pull/10854/files#diff-2941c4dbb172f73b42807fa33f8fc742R294